src/pages/plugins/index.js

```diff
@@ -26,6 +27,11 @@ class Plugin {
     return this._package.name;
   }
 
+  get deprecatedMessage() {
+    const deprecatedMessage= this._flags.deprecated ? this._flags.deprecated : false;
```

Comment:
Don't use `false` when the other value is a string. Instead, use an empty string or undefined.

Reply:
@donghwipark you never fixed this.